### PR TITLE
Core/new header docs

### PR DESCRIPTION
**Pull request by clairesunstudio**
Render new header and header hamburger in core documentation

### CODE REVIEW

File: packages/core/stories/components/Header/Header.stories.js [hunk 1]
```diff
@@ -91,240 +94,16 @@ const headerUtilSearch = (
...
-                type: 'external'
-              }
-            ]
-          },
-          text: 'Log in to...'
-        }
-      ]
-    }}
+    mainItems={mainNavData}
+    utilityItems={[]}
```
Comment (smurrayatwork) [inline packages/core/stories/components/Header/Header.stories.js:98]: Set UtilityNav to null here:
```suggestion
    UtilityNav={null}
```

File: packages/core/stories/components/Header/Header.stories.js [hunk 2]
```diff
@@ -91,240 +94,16 @@ const headerUtilSearch = (
...
+  />
+);
+
+
+const headerHamburger = (
+  <HeaderHamburger
+    mainItems={mainNavData}
+    utilityItems={[]}
```
Comment (smurrayatwork) [inline packages/core/stories/components/Header/Header.stories.js:105]:
```suggestion
    UtilityNav={null}
```

File: packages/core/stories/components/Header/Header.stories.js (hunk 1 above)
Comment (clairesunstudio) [reply packages/core/stories/components/Header/Header.stories.js:98]: won't setting it to null remove the utility nav items? I'd like to show the utility nav example same what's live on mass.gov

Comment (smurrayatwork) [reply packages/core/stories/components/Header/Header.stories.js:98]: Oh! No you're right, disregard.

**Review by smurrayatwork — APPROVED** [review]

**Pull request merged**